scipy/optimize/_shgo.py

```diff
@@ -1355,8 +1355,8 @@ def minimize(self, x_min, ind=None):
 
         # Local function evals for all minimizers
         self.res.nlfev += lres.nfev
-        if 'njev' in lres:
-            self.res.nljev += lres.njev
+        if 'ngev' in lres:
```

Comment:
SHGO is still having problems with this change. I skipped some tests and I'll come back to it.